routes/kingdoms.js

```diff
@@ -0,0 +1,107 @@
+const express = require('express');
+const router = express.Router();
+const { getKingdoms, getCastles, getLieges, getVassals } = require('../services/kingdom_getters');
```

Comment:
Interesting organization. Another way to separate this would be to have a module per entity—rather than grouping modules by operation as you've done. Just something to think about, there is nothing wrong w/ this approach :)